src/components/editor/List.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactList from 'react-list';
+import { Link } from 'react-router';
+
+export default class List extends React.Component {
+  constructor(props) {
+    super(props);
+    this.renderItem = this.renderItem.bind(this);
+  }
+  renderItem(index, key) {
+    index = this.props.indexConverter[index];
+    return (
+      <div key={key}>
+        <Link to={this.props.link + this.props.data[index].slug + "/"}>{this.props.data[index].title}</Link>
+      </div>
+    );
+  }
+  render() {
+    return (
+      <div style={this.props.style}>
+        {this.props.name ? this.props.name : null}
+        <div style={{overflow: 'auto', maxHeight: this.props.maxHeight, maxWidth: this.props.maxWidth}}>
+          <ReactList
```

Comment:
What do you mean? 

That we shouldn't use the react-list component and just use a normal non-scrolling list? I guess that could also be just fine, I guess it's also a trailing idea before I decided to page it, I thought I was just going to scroll. 

It's nice if you're looking at it on a small screen though I guess, because then no matter what the amount of elements you put into the list will fit, as then it will just scroll.